Add HumanWorker, OTTO Robots, RGL and UR Robots robotic Gems (#2517)
o3de/o3de.org: content/docs/user-guide/gems/reference/robotics/humanworker.md

---
linkTitle: Human Worker
title: Human Worker Gem
description: The Human Worker Gem provides a collection of animated human worker assets that can be used in robotic simulations with Robot Operating System (ROS) 2 in Open 3D Engine (O3DE).
toc: true
---

<!-- # Human Worker Gem -->

> The Human Worker Gem provides a collection of animated human worker assets that can be used in robotic simulations with [ROS&nbsp;2 Gem](./ros2.md) in **Open 3D Engine (O3DE)**. The human worker can navigate between the waypoints in the scene and is visible to any robots that maneuver in the same area. It uses 

Reviewer: ```suggestion
This Gem provides a simulated human worker who can navigate between waypoints in the scene and is visible to any robots that maneuver in the same area.
```

This gem contains the following components:
* Human worker mesh with textures
* Mesh animations of human standing
* Mesh animations of human walking
* O3DE components for basic navigation of the human worker in a scene

> An example of its use can be found in [ROSCon2023Demo](https://github.com/RobotecAI/ROSCon2023Demo) project. The Gem is available from our [Github repository](https://github.com/RobotecAI/o3de-humanworker-gem). For more information, refer to the _README_ file of the Gem.

Reviewer: ```suggestion
You can find an example of this Gem use in [ROSCon2023Demo](https://github.com/RobotecAI/ROSCon2023Demo). The Gem project has a [GitHub repository](https://github.com/RobotecAI/o3de-humanworker-gem). For more information, refer to the _README_ file of the Gem.
```
